native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -0,0 +1,297 @@
+    minHeight: spacingXl - (1 * 2),
+  },
+  bannerContainerPadded: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingBottom: spacingSm,
+    paddingLeft: spacingMd,
+    paddingRight: spacingMd,
+    paddingTop: spacingSm,
+  },
+  bannerContainerPaddedDismissable: {
+    paddingRight: 0,
+  },
+  closeButtonContainer: {
+    height: '100%',
+    width: ((2 * spacingMd) + spacingBase),
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  expandedChildContainer: {
+    paddingBottom: spacingSm,
+    paddingLeft: spacingMd,
+    paddingRight: spacingMd,
+  },
+  outerContainerSuccess: {
+    borderColor: colorGreen500,
+  },
+  outerContainerWarn: {
+    borderColor: colorYellow500,
+  },
+  outerContainerError: {
+    borderColor: colorRed500,
+  },
+  text: {
+    flex: 1,
+    flexGrow: 1,
+    paddingLeft: spacingSm,
+  },
+  icon: {
+    width: spacingBase,
+    height: spacingBase,
+  },
+  iconSuccess: {
+    tintColor: colorGreen500,
+  },
+  iconWarn: {
+    tintColor: colorYellow500,
+  },
+  iconError: {
+    tintColor: colorRed500,
+  },
+  button: {
+    tintColor: colorGray700,
+    width: spacingBase,
+    height: spacingBase,
+  },
+  buttonClose: {
+    tintColor: colorGray500,
+  },
+});
+
+const BpkBannerAlert = (props) => {
+  const { type, message, onPress, dismissable, expanded, actionButtonLabel, style, children, ...rest } = props;
+
+  const expandable = children !== null;
+
+  let iconSource = null;
+  let buttonIconSource = null;
+
+  const outerStyleFinal = [styles.outerContainer];
+  const iconStyle = [styles.icon];
+  const buttonIconStyle = [styles.icon, styles.button];
+  const expandedChildContainer = [styles.expandedChildContainer];
+
+  if (style) {
+    outerStyleFinal.push(style);
+  }
+
+  if (type === ALERT_TYPES.SUCCESS) {
+    iconSource = tickIcon;
+    outerStyleFinal.push(styles.outerContainerSuccess);
+    iconStyle.push(styles.iconSuccess);
+  } else if (type === ALERT_TYPES.WARN) {
+    iconSource = informationIcon;
+    outerStyleFinal.push(styles.outerContainerWarn);
+    iconStyle.push(styles.iconWarn);
+  } else if (type === ALERT_TYPES.ERROR) {
+    iconSource = informationIcon;
+    outerStyleFinal.push(styles.outerContainerError);
+    iconStyle.push(styles.iconError);
+  }
+
+  if (dismissable) {
+    buttonIconSource = closeIcon;
+    buttonIconStyle.push(styles.closeIcon);
+  } else if (children !== null) {
+    if (expanded) {
+      buttonIconSource = chevronUpIcon;
+    } else {
+      buttonIconSource = chevronDownIcon;
+    }
+  }
+
+  const iconComponent = (
+    <Image
+      style={iconStyle}
+      source={iconSource}
+    />
+  );
+
+  const textComponent = (
+    <BpkText textStyle="sm" style={styles.text}>{message}</BpkText>
+  );
+
+  const actionComponent = (
+    <Image
+      style={buttonIconStyle}
+      source={buttonIconSource}
+    />
+  );
```

Comment:
Could this be made to use `react-native-bpk-component-icon`?

Reply:
Not sure - I think we should discuss this at standup, and then create a PR to replace all PNGs with the icon component